modules/local/omextractor/meta.yml

```
@@ -0,0 +1,33 @@
+name: omextractor
+description: Extract OME tags and store them into a xml file.
+keywords:
+  - OME tags
+  - bioinformatics tools
+tools:
+  - bftools:
+      description: |
+        Suite of tools to handle several imaging protocols.
+      homepage: https://www.openmicroscopy.org/bio-formats/
+      documentation: https://www.openmicroscopy.org/bio-formats/
+      licence: ["GPL-3.0-or-later"]
+      identifier: biotools:multiqc
+input:
+  - - image:
```

The `input` block is missing the `meta` part of the tuple. I think it should look like this:
```suggestion
  - - meta:
        type: map
        description: |
          Groovy Map containing sample information
          e.g. [ id:'test' ]
    - image:
```